server/middleware/auth.js

```diff
@@ -1,10 +1,23 @@
+const REDIRECT_ROUTES = {
+  home: '/',
+  main: '/todos'
+};
+
+
 module.exports = {
-    ensureAuth: function (req, res, next) {
-      if (req.isAuthenticated()) {
-        return next()
-      } else {
-        res.redirect('/')
-      }
+  ensureAuth: function (req, res, next) {
+    if (req.isAuthenticated()) {
+      return next()
+    } else {
+      res.redirect(REDIRECT_ROUTES.home)
+    }
+  },
+  ensureGuest: function (req, res, next) {
+    if (!req.isAuthenticated()) {
+      return next()
+    } else {
+      res.redirect(REDIRECT_ROUTES.main) 
```

Comment:
Just hard code these redirects to "/" for now, until we setup routes on the React side.